### ISSUE TITLE
Schema is undefined, even after successful introspection query

### ISSUE THREAD

**Issue by rmosolgo** [graphql/graphiql#16]
I'm trying to hook this up to an automatically generated Ruby backend, so I'm almost sure that the root cause is on my end, but I'm not sure what else to check :)

I've cloned the repo & built the example, then copied `graphql.min.js`, `graphql.css`, and `index.html` into my own project. I tweaked the fetcher a bit to point at my endpoint and include cookies. 

At this point, when I load the page, GraphiQL makes its request for the schema, and I can see the expected result coming back:

![image](https://cloud.githubusercontent.com/assets/2231765/9507906/48d8f820-4c06-11e5-920d-326eca6648fc.png)
_([full dump](https://gist.github.com/rmosolgo/2ff667d279bed0eef3eb))_

However, I don't get any type hinting, and when I try to submit a query, I get an error: 

![image](https://cloud.githubusercontent.com/assets/2231765/9507492/6e24ab36-4c03-11e5-9de5-84ebc21817f8.png)

I can see it's coming from here: 

![image](https://cloud.githubusercontent.com/assets/2231765/9507799/5ee87628-4c05-11e5-9da4-2fdea3c0c321.png)

Which is here: https://github.com/graphql/graphql-js/blob/81a7d7add03adbb14dc852bbe45ab030c0601489/src/utilities/TypeInfo.js#L125-L127

So ... somehow that local variable `schema` is undefined. I know I'm a long way down a lonely road, but do you have any suggestion what I can check next? Do you think my endpoint is responding properly? Is there something else I can test?

**Comment by leebyron** [comment]
Can you use the non minified source to get some better stacks?

**Comment by rmosolgo** [comment]
definitely, how can I get it?

**Comment by asiandrummer** [comment]
When you build the example there should be `graphiql.js` in the main `graphiql/` directory. Or you could do `npm run build` in the main graphiql directory as well.

**Comment by rmosolgo** [comment]
Thanks for your suggestion! I found the unminified version and used "Pause on Caught Exceptions" to track it down. 

I was using a stand-in for my mutationType and it had no `fields` key in the JSON. Sure enough, there was an exception in there:

> Mutation fields must be an object with field names as keys or a function which returns such an object.

But I guess it was caught somewhere along the way. 

Anyways, I removed my dummy mutation type & then :tada:  

Thanks for your help!
